mmv1/templates/terraform/constants/resource_dns_resource_record_set.go.erb

```diff
@@ -11,9 +18,28 @@ func rrdatasDnsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return false
 }
 
-func ipv6AddressforDnsDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
-	oldIp := net.ParseIP(old)
-	newIp := net.ParseIP(new)
-
-	return oldIp.Equal(newIp)
+func ipv6AddressforDnsDiffSuppress(x, y []string, _ *schema.ResourceData) bool {
+	// compare two lists of unordered IPs
```

Comment:
Can we compare the length of the two arrays early on and return false if they are different? That's an easy way to short-circuit this whole method

Reply:
I knew this is a common practice in the suppress functions. But considering if there are record dups in old or new that could lead to a false result.

Reply:
Ah ok. Is it possible for there to be duplicates in the set that will parse to the same IP? In that case leave it how it is